README.rst

```diff
@@ -66,7 +66,7 @@ Contributing
 ------------
 
 We welcome contributions. Please follow these
-`guidelines <https://github.com/quantumlib/cirq/blob/master/CONTRIBUTING.md>`__.
+`guidelines <https://github.com/quantumlib/cirq/blob/master/dev_tools/CONTRIBUTING.md>`__.
```

Comment:
I'm not sure if there are legal reasons for CONTRIBUTING to have to be at the top level or not. I feel uncomfortable moving this file.